code/modules/food_and_drinks/food/snacks/meat.dm

```diff
@@ -190,6 +190,23 @@
 	icon_state = "meatwheat_clump"
 	bitesize = 4
 
+/obj/item/weapon/reagent_containers/food/snacks/meat/rawbacon
+	name = "raw piece of bacon"
+	desc = "A raw piece of bacon."
+	icon_state = "bacon"
+	cooked_type = /obj/item/weapon/reagent_containers/food/snacks/meat/rawbacon/bacon
+	bitesize = 2
+	list_reagents = list("nutriment" = 1)
+	filling_color = "#B22222"
+	var/meat_type = "meat"
+
+/obj/item/weapon/reagent_containers/food/snacks/meat/rawbacon/bacon
```

Comment:
since bacon is a child of raw bacon, cooking cooked bacon will produced cooked bacon.... I might be best to do meat/rawbacon and meat/bacon, like cutlets do.